rest-api/src/test/java/gov/cms/qpp/conversion/api/services/CpcFileServiceImplTest.java

```diff
@@ -104,6 +106,41 @@ void testGetFileByIdNoFile() throws IOException {
 		assertThat(expectedException).hasMessageThat().isEqualTo(CpcFileServiceImpl.FILE_NOT_FOUND);
 	}
 
+	@Test
+	void testProcessFileByIdSuccess() {
+		when(dbService.getMetadataById(anyString())).thenReturn(buildFakeMetadata(true, false));
+
+		String message = objectUnderTest.processFileById(MEEP);
+
+		verify(dbService, times(1)).getMetadataById(anyString());
```

Comment:
I would verify that it calls `getMetadataById ` with `MEEP` as the argument instead of `anyString()`.

Reply:
👍